spark-cassandra-connector-embedded/src/main/scala/com/datastax/spark/connector/embedded/EmbeddedCassandra.scala

```diff
@@ -72,8 +72,13 @@ private[connector] class CassandraRunner(val configTemplate: String) extends Emb
       copyTextFileWithVariableSubstitution(input, output, properties)
     }
   }
+  private def makeJarPath() : String ={
```

Comment:
Don't add parens if there are no args and it's not fu(): Unit = {}

private def makeJarPath: String = {